File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    let answer = 0;
    score.sort();
    
    for(let i = score.length - 1; i >= m - 1; i -= m){
        answer += score[i - m + 1] * m;
    }
    
    return answer;
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(int k, int m, vector<int> score) {
    int answer = 0;
    sort(score.begin(), score.end());
    for(int i = score.size() - 1; i > 0; i -= m){
        int min = score[i];
        for(int j = i - m + 1; j < i; j++){
            if(min > score[j]) min = score[j];
        }
        
        answer += min * m;
    }
    
    return answer;
}